app/views/posts/index.html.erb

```diff
@@ -1 +1,33 @@
-<h1>Here is a list of all posts by user 1</h1>
\ No newline at end of file
+<article class="main-article" style="width: 90%; margin: auto">
+  <div style="display: flex; align-items: center; justify-content: space-between;">
+    <%= image_tag @user.photo, :class => "user-avatar" %>
+    <div class="user-details">
+      <p><%= @user.name %></p>
+      <span>Number of posts: <%= @user.post_counter %></span>
+    </div>
+  </div>
+
+  <% @user.recent_posts.each_with_index do |post, index| %>
+    <div>
+      <div class="post">
```

Comment:
- [ ] [Optional] Nice work on displaying the post information! 👏 However, it would be beneficial to link each post to its show page. This way, clicking on each post would direct users to the respective show page. Adding these links will enhance the user experience and align with common navigation practices. Consider implementing this improvement to make your application more user-friendly. Keep up the good work! 🚀🔗